Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -52,152 +55,182 @@ public class TwineNode : MonoBehaviour
-            for (int index = 0; index < this.validLinkNames.Length; index++)
-            {
-                if (GUILayout.Button(this.validLinkNames[index]))
-                {
-                    this.ActivateChildAtIndex(index);
-                }
-            }
+            FirstPersonInteractor interactor = (FirstPersonInteractor)FindObjectOfType(typeof(FirstPersonInteractor));
+            GameObject interactorObject = interactor.gameObject;
+            this._Activate(interactorObject);
+        }
+            StartCoroutine(Example());
+    }
 
-            GUI.EndGroup();
+    IEnumerator Example()
+    {
+        if (this.children.Length == 0) {
+            yield return new WaitForSeconds(5);
+            this.Deactivate();
         }
-        else if (fanfold) //Draw a bunch of boxes
-        {
-            float frameWidth = Math.Min(Screen.width / 3, 150);
-            float frameHeight = Math.Min(Screen.height / 2, 500);
-            int index = TwineNodeList.IndexOf(this);
-            Rect frame = new Rect(10 + index * 150, 10, frameWidth, frameHeight);
-            GUI.BeginGroup(frame);
-            GUIStyle style = new GUIStyle(GUI.skin.box);
-            style.wordWrap = true;
-            style.fixedWidth = frameWidth;
-            GUILayout.Box(this.content, style);
-
-            GUI.EndGroup();
+    }
 
+    void Update ()
+    {
+//        foreach (TwineNode node in TwineNodeList){
+//            print(node.name + " " + TwineNodeList.IndexOf(node));
+//            print(visibleNodeIndex);
+//        }
+            UpdateConditionalLinks();
+            if (this.enabled) {
+                if (Input.GetKeyDown (KeyCode.C) && TwineNodeList.IndexOf(this) == 0 && allMinimized == false){
+                    fanfold = !fanfold;
+                }
+                if (Input.GetKeyDown (KeyCode.M) && TwineNodeList.IndexOf(this) == 0){
+                    allMinimized = !allMinimized;
+                }
+                if (!TwineNodeList.Contains(this)){
+                    TwineNodeList.Add(this);
+                }
+                if (Input.GetKeyDown (KeyCode.Tab) && TwineNodeList.IndexOf(this) == 0 && allMinimized == false && fanfold == false){
+                    if (visibleNodeIndex == TwineNodeList.Count - 1) {
+                        print("1");
+                        visibleNodeIndex = 0;
+                    }
+                    else {
+                        print("2");
+                        visibleNodeIndex++;
+                    }
+                }
+                if (TwineNodeList.IndexOf(this) == visibleNodeIndex) {
+                    this.isMinimized = false;
+                }
+                else {
+                    this.isMinimized = true;
+                }
+                if (this.isDecisionNode) {
+                    this.isOptionsGuiOpen = true;
+                } else if (this.isConditionNode) {
+                    // get the $color value from global list
```

Comment:
This comment seems to be specific to your own testing scene and story. There is not necessarily a $color value. Remove this comment or replace with a more general comment.